docs/source/release-notes/1.17.rst

```diff
@@ -135,6 +135,35 @@ Upgrading
 All bits all need to be rerun because of model changes. This can be done on the
 organization settings page.
 
+Running the Origin Migration
+================================
+Upon upgrading, one migration needs to be triggered manually.
+This is the `boefjes/tools/upgrade_v1_16_0.py` script, which you can run in your ``boefje`` environment using:
+
+.. code-block:: sh
+
+    python -m tools.upgrade_v1_16_0
+
+Or more concretely, using Docker:
+
+.. code-block:: sh
+
+    docker compose exec boefje python -m tools.upgrade_v1_16_0
```

Comment:
I suggest using `docker compose run --rm` instead of `exec`. Using `run --rm` will ensure the script executes whether or not the `boefje` service is already running, as it creates a new instance and cleans it up after running

```suggestion
    docker compose run --rm boefje python -m tools.upgrade_v1_16_0
```